swagger.yaml

```diff
@@ -3153,6 +3153,48 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
+  baremetal_platform:
+    description: Baremetal platform specific configuration upon which to perform the installation
+    type: object
+    properties:
+      provisioningNetwork:
+        type: string
+        enum: [Disabled, Enabled]
+
+  vsphere_platform:
+    type: object
+    description: Vsphere platform specific configuration upon which to perform the installation
+    required:
+      - username
+      - password
```

Comment:
all those params are required to make it work? or they are required only n day2?

Reply:
For day1 I don't think we need any  vsphere credentials